contrib/exporters/README.md

```diff
@@ -0,0 +1,190 @@
+# Quick Setup
```

Comment:
Can an introduction be added here to describe who this exporter setup is intended for? It also might be good for this to be moved into the `zcashd` book, rather than living in the metrics section; this `README.md` could even be a symbolic link to a document in the book, if that would make it more easily findable.

It'd also be nice to have brief description of what role each component serves. For example, I don't know what `loki` is for.

Reply:
This is intended for any zcash user or node operator really. Not sure how 6167 makes that more clear.
Loki is a standard logging tool used with Prometheus.